src/main/resources/META-INF/native-image/reflect-config.json

```diff
@@ -0,0 +1,233 @@
+    "allPublicMethods": true,
+    "allDeclaredConstructors":true
+  },
+  {
+    "name":"io.lettuce.core.api.sync.RedisStreamCommands",
+    "allPublicMethods": true,
+    "allDeclaredConstructors":true
+  },
+  {
+    "name":"io.lettuce.core.api.sync.RedisStringCommands",
+    "allPublicMethods": true,
+    "allDeclaredConstructors":true
+  },
+  {
+    "name":"io.lettuce.core.api.sync.RedisTransactionalCommands",
+    "allPublicMethods": true,
+    "allDeclaredConstructors":true
+  },
+
+  {
+    "name":"io.lettuce.core.cluster.api.sync.RedisClusterCommands",
+    "allPublicMethods": true,
+    "allDeclaredConstructors":true
+  },
+
+  {
+    "name":"io.lettuce.core.protocol.CommandHandler",
+    "allPublicMethods": true,
+    "allDeclaredConstructors":true
+  },
+  {
+    "name":"io.lettuce.core.protocol.ConnectionWatchdog",
+    "allPublicMethods": true,
+    "allDeclaredConstructors":true
+  },
+
+  {
+    "name":"io.lettuce.core.resource.ClientResources",
+    "allPublicMethods": true,
+    "allDeclaredConstructors":true
+  },
+
+  {
+    "name":"io.lettuce.core.resource.DefaultClientResources",
+    "allDeclaredFields" : true,
+    "allPublicMethods": true,
+    "allDeclaredConstructors":true
+  },
+  {
+    "name":"io.netty.buffer.AbstractByteBufAllocator",
+    "allPublicMethods": true,
+    "allDeclaredFields":true,
+    "allDeclaredMethods":true,
+    "allDeclaredConstructors":true
+  },
+  {
+    "name":"io.netty.buffer.PooledByteBufAllocator",
+    "allPublicMethods": true,
+    "allDeclaredFields":true,
+    "allDeclaredMethods":true,
+    "allDeclaredConstructors":true
+  },
+  {
+    "name":"io.netty.channel.ChannelDuplexHandler",
+    "allPublicMethods": true,
+    "allDeclaredConstructors":true
+  },
+  {
+    "name":"io.netty.channel.ChannelHandlerAdapter",
+    "allPublicMethods": true,
+    "allDeclaredConstructors":true
+  },
+  {
+    "name": "io.netty.channel.ChannelInboundHandlerAdapter",
+    "allPublicMethods": true,
+    "allDeclaredConstructors":true
+  },
+  {
+    "name": "io.netty.channel.ChannelInitializer",
+    "allPublicMethods": true,
+    "allDeclaredConstructors":true
+  },
+  {
+    "name": "io.netty.channel.ChannelOutboundHandlerAdapter",
+    "allPublicMethods": true,
+    "allDeclaredConstructors":true
+  },
+  {
+    "name": "io.netty.channel.DefaultChannelPipeline$HeadContext",
+    "allPublicMethods": true,
+    "allDeclaredConstructors":true
+  },
+  {
+    "name": "io.netty.channel.DefaultChannelPipeline$TailContext",
+    "allPublicMethods": true,
+    "allDeclaredConstructors":true
+  },
+  {
+    "name": "io.netty.channel.socket.nio.NioSocketChannel",
+    "allPublicMethods": true,
+    "allDeclaredConstructors":true
+  },
+  {
+    "name": "io.netty.handler.codec.MessageToByteEncoder",
+    "allPublicMethods": true,
+    "allDeclaredConstructors":true
+  },
+  {
+    "name":"io.netty.util.ReferenceCountUtil",
+    "allPublicMethods": true,
+    "allDeclaredConstructors":true
+  }
+]
```

Comment:
Additional types (for latency tracking, since we do class presence checks)

* org.HdrHistogram.Histogram
* org.LatencyUtils.PauseDetector
* rx.Completable
* reactor.core.publisher.Mono
* io.reactivex.Flowable
* io.reactivex.rxjava3.core.Flowable